dist/ramda.js

```diff
@@ -3263,6 +3246,35 @@
 
     /**
      * Converts an object into an array of key, value arrays.
+     * Only the object's own properties are used.
+     * Note that the order of the output array is not guaranteed to be
+     * consistent across different JS platforms.
+     *
+     * @func
+     * @memberOf R
+     * @category Object
+     * @sig {String: *} -> [[String,*]]
+     * @param {Object} obj The object to extract from
+     * @return {Array} An array of key, value arrays from the object's own properties.
+     * @example
+     *
+     *      R.toPairs({a: 1, b: 2, c: 3}); //=> [['a', 1], ['b', 2], ['c', 3]]
+     */
+    var toPairs = _curry1(function toPairs(obj) {
+        var pairs = [];
+        for (var prop in obj) {
+            if (Object.prototype.hasOwnProperty.call(obj, prop)) {
```

Comment:
All the activity and I've been offline for days!

And I don't have much time now, unfortunately.  I just want to make a few points:

* I don't think #829 is a good reason for changing our implementations.  I know that @davidchambers has made a number of posts with this rationale, and I simply don't buy it.  I think there are good reasons to ensure that our dependency trees are not horribly deep the way that lodash used to be before a recent refactoring.  But I don't see a reason for every function to try to stand alone either.  If the changes help for other reasons, I have no objections, but let's not make a fetish out of this.  Any `require` mechanism will straighten that out in a reasonable way.  (On a somewhat related note, I _would_ like to remove the separation between `internal/_xyz` and `xyz` where we can.  I think that unless there are really strong reasons for not doing so, we should simply use the curried, dispatched functions internally.)

* The reason people capture `Object.prototype.hasOwnProperty` is generally not because someone might override it on `Object.prototype` but that they might do so on an arbitrary object.  Obviously if they've done it on the global before you take your copy, you're already out of luck, so there's not much use in worrying about that.  But for other objects, if you capture the global version and use that one instead, you won't be caught out by someone's screwy object.  This is not a major concern for me, but it's a small enough effort to ensure that we don't get caught in these circumstances.

* Is there any major reason for this PR besides reducing internal dependencies?  Do you see this as cleaner code?  More performant?  I'm hesitant to do anything base solely upon the dependency tree.